CODE_OF_CONDUCT.md

```diff
@@ -39,3 +37,19 @@ CoC](http://www.rust-lang.org/conduct.html) as well as the
 * Avoid the use of personal pronouns in code comments or
   documentation. There is no need to address persons when explaining
   code (e.g. "When the developer").
+
+### Contact
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by:
+
+* Emailing [report@nodejs.org](mailto:report@nodejs.org) (this will email all TSC members)
+* Contacting [individual TSC members](https://nodejs.org/en/foundation/tsc/#current-members-of-the-technical-steering-committee).
```

Comment:
This is just explicitly repeating the contact guidelines in the [moderation policy](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#requesting-moderation).